chainntnfs/interface_test.go

```diff
@@ -1533,6 +1533,31 @@ func testCatchUpOnMissedBlocksWithReorg(miner1 *rpctest.Harness,
 		t.Fatalf("unable to join node on blocks: %v", err)
 	}
 
+	// The two should be on the same block hash.
+	timeout := time.After(10 * time.Second)
+	for {
+		nodeHash1, _, err := miner1.Node.GetBestBlock()
+		if err != nil {
+			t.Fatalf("unable to get current block hash: %v", err)
+		}
+
+		nodeHash2, _, err := miner2.Node.GetBestBlock()
+		if err != nil {
+			t.Fatalf("unable to get current block hash: %v", err)
+		}
+
+		if *nodeHash1 != *nodeHash2 {
+			select {
+			case <-timeout:
+				t.Fatalf("Unable to sync two chains")
+			default:
```

Comment:
to avoid spamming the nodes w/ `GetBestBlock` while they sync, we can add a small timeout here before continuing, ala:
```golang
...
case <-time.After(50 * time.Millisecond):
        continue
```

Reply:
fixed :)